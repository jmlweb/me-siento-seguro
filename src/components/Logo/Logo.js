import React from 'react';

import styles from './Logo.module.css';
import logoImage from './logo.png';

const Logo = () => (
  <div className={styles.wrapper}>
    <span className={styles.title}>GiphySearch</span>
    <img className={styles.logo} src={logoImage} alt="" />
    <span>by Sewan</span>
  </div>
);

export default Logo;
