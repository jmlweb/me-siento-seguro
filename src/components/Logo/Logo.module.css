.wrapper {
  display: flex;
  justify-content: space-between;
  align-items: center;
  width: 100%;
}

.logo {
  display: block;
  margin-right: 10px;
  filter: invert(100%);
  width: 188px;
}

.title {
  font-size: 22px;
  text-transform: uppercase;
}
