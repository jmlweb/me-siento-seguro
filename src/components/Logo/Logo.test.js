import React from 'react';
import { shallow } from 'enzyme';

import Logo from './Logo';

describe('Logo', () => {
  test('component renders', () => {
    const element = <Logo />;
    const wrapper = shallow(element);
    expect(wrapper).toBeTruthy();
  });
});
