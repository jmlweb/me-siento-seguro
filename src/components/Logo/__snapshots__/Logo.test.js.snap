// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo snapshot 1`] = `
<div
  className="wrapper"
>
  <span
    className="title"
  >
    GiphySearch
  </span>
  <img
    alt=""
    className="logo"
    src="logo.png"
  />
  <span>
    by Sewan
  </span>
</div>
`;
